starknet_py/tests/e2e/conftest.py

```diff
@@ -33,6 +37,20 @@
     "0x60D7C88541F969520E46D39EC7C9053451CFEDBC2EEB847B684981A22CD452E"
 )
 
+INTEGRATION_NEW_ACCOUNT_PRIVATE_KEY = "0x1"
+
+INTEGRATION_NEW_ACCOUNT_ADDRESS = (
+    "0X126FAB6AE8ACA83E2DD00B92F94F3402397D527798E18DC28D76B7740638D23"
+)
+
+
+@pytest.fixture(scope="module")
+def event_loop():
+    policy = asyncio.get_event_loop_policy()
+    loop = policy.new_event_loop()
+    yield loop
+    loop.close()
```

Comment:
I guess we could add a comment here that this was added because the problems with async fixtures in case we run into problems with them in the future.